routing/road_access.hpp

```diff
@@ -0,0 +1,29 @@
+#pragma once
+
+#include <cstdint>
+#include <vector>
+
+namespace routing
+{
+// This class provides information about road access classes.
+// For now, only restrictive types (such as barrier=gate and access=private)
+// and only car routing are supported.
+class RoadAccess final
+{
+public:
+  std::vector<uint32_t> & GetPrivateRoads() { return m_privateRoads; }
```

Comment:
Why do we need this version of GetPrivateRoads()? Why not to load private roads and then set them via SetPrivateRoads?